pages/settingspage.py

```diff
@@ -137,16 +130,24 @@ def save_settings(self):
                     "onlybuyone":         self.buy_one_checkbox.isChecked(),
                     "dont_buy":           self.dont_buy_checkbox.isChecked(),
                     "random_delay_start": self.random_delay_start.text(),
-                    "random_delay_stop":  self.random_delay_stop.text()}
+                    "random_delay_stop":  self.random_delay_stop.text(),
+                    "target_user": self.target_user_edit.text(),
+                    'target_pass': Encryption().encrypt(self.target_pass_edit.text()).decode("utf-8")}
+
         write_data("./data/settings.json",settings)
         self.update_settings(settings)
         QtWidgets.QMessageBox.information(self, "Phoenix Bot", "Saved Settings")
 
     def update_settings(self, settings_data):
-        global webhook, webhook_on_browser, webhook_on_order, webhook_on_failed, browser_on_failed, dont_buy, random_delay_start, random_delay_stop
+        global webhook, webhook_on_browser, webhook_on_order, webhook_on_failed, browser_on_failed, dont_buy, random_delay_start, random_delay_stop, target_user, target_pass
         settings.webhook, settings.webhook_on_browser, settings.webhook_on_order, settings.webhook_on_failed, settings.browser_on_failed, settings.buy_one, settings.dont_buy = settings_data["webhook"], settings_data["webhookonbrowser"], settings_data["webhookonorder"], settings_data["webhookonfailed"], settings_data["browseronfailed"], settings_data['onlybuyone'], settings_data['dont_buy']
 
         if settings_data.get("random_delay_start", "") != "":
             settings.random_delay_start = settings_data["random_delay_start"]
         if settings_data.get("random_delay_stop", "") != "":
             settings.random_delay_stop = settings_data["random_delay_stop"]
+        if settings_data.get("target_user_edit", "") != "":
+            settings.target_user = settings_data["target_user"]
+        if settings_data.get("target_pass_edit", "") != "":
+            settings.target_pass = settings_data["target_pass"]
```

Comment:
Bug: Current code doesn't set settings.target_user and settings.target_pass correctly.

```suggestion
        if settings_data.get("target_user", "") != "":
            settings.target_user = settings_data["target_user"]
        if settings_data.get("target_pass", "") != "":
            settings.target_pass = (Encryption().decrypt(settings_data["target_pass "].encode("utf-8"))).decode("utf-8")
```